packages/paste-core/components/button/type-docs.json

```diff
@@ -0,0 +1,1713 @@
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "placeholder": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "prefix": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "pressed": {
+      "type": "boolean",
+      "defaultValue": false,
+      "required": false,
+      "externalProp": false,
+      "description": "Sets the aria-pressed attribute. Must be used with 'secondary' or 'secondary_icon' variants."
+    },
+    "property": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "radioGroup": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "resource": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "results": {
+      "type": "number",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "role": {
+      "type": "AriaRole",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "security": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "size": {
+      "type": "ButtonSizes",
+      "defaultValue": "default",
+      "required": false,
+      "externalProp": false
+    },
+    "slot": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "spellCheck": {
+      "type": "Booleanish",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "suppressContentEditableWarning": {
+      "type": "boolean",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "suppressHydrationWarning": {
+      "type": "boolean",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "tabIndex": {
+      "type": "ButtonTabIndexes",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": false
+    },
+    "target": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": false
+    },
+    "title": {
+      "type": "string",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "translate": {
+      "type": "\"yes\" | \"no\"",
+      "defaultValue": null,
+      "required": false,
+      "externalProp": true
+    },
+    "type": {
+      "type": "ButtonTypes",
+      "defaultValue": "button",
+      "required": false,
+      "externalProp": false,
+      "description": "If the Button is inside a <form>: use 'submit'. Otherwise use 'button' (default)."
```

Comment:
issue: angle brackets are causing bad formatting on the docs page

Reply:
Yeah, unescaped HTML will do that to a page. Needs backticks